code/game/objects/items/RWD.dm

```diff
@@ -0,0 +1,261 @@
+/obj/item/rwd
+	name = "rapid wiring device"
+	desc = "A device used to rapidly lay cable & pick up stray cable pieces laying around."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "rcl-0"
+	inhand_icon_state = "rcl-0"
+	opacity = FALSE
+	force = 5 //Plastic is soft
+	throwforce = 5
+	throw_speed = 1
+	throw_range = 7
+	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
should be bulky

Reply:
I dont think it needs to be bulky its essentially the same as having a box in your backpack that you fill with cable coils

Reply:
Code copied from rlc so makes sense it should be the same weight class as that